miso-service/src/main/java/uk/ac/bbsrc/tgac/miso/service/impl/DefaultSampleService.java

```diff
@@ -381,7 +381,6 @@ private DetailedSample findOrCreateParent(DetailedSample sample) throws IOExcept
   private SampleTissue createParentTissue(SampleTissue tissue, DetailedSample child) throws IOException {
     log.debug("Creating a new Tissue to use as a parent.");
     tissue.setProject(child.getProject());
-    tissue.setDescription("Tissue");
```

Comment:
Want to make a separate ticket for making better use of the 'isSynthetic' field? I don't think storing the same information in two places - one of which is freely-editable free text (description) - is the solution